release-notes/9.0/preview/preview6/api-diff/Microsoft.NETCore.App/9.0-preview6_System.Runtime.Intrinsics.Arm.md

```diff
@@ -0,0 +1,1058 @@
++            public unsafe static void StoreVectorAndZip(long* address, (Vector128<long> Value1, Vector128<long> Value2, Vector128<long> Value3, Vector128<long> Value4) value);
++            public unsafe static void StoreVectorAndZip(long* address, (Vector128<long> Value1, Vector128<long> Value2, Vector128<long> Value3) value);
++            public unsafe static void StoreVectorAndZip(long* address, (Vector128<long> Value1, Vector128<long> Value2) value);
++            public unsafe static void StoreVectorAndZip(sbyte* address, (Vector128<sbyte> Value1, Vector128<sbyte> Value2, Vector128<sbyte> Value3, Vector128<sbyte> Value4) value);
++            public unsafe static void StoreVectorAndZip(sbyte* address, (Vector128<sbyte> Value1, Vector128<sbyte> Value2, Vector128<sbyte> Value3) value);
++            public unsafe static void StoreVectorAndZip(sbyte* address, (Vector128<sbyte> Value1, Vector128<sbyte> Value2) value);
++            public unsafe static void StoreVectorAndZip(float* address, (Vector128<float> Value1, Vector128<float> Value2, Vector128<float> Value3, Vector128<float> Value4) value);
++            public unsafe static void StoreVectorAndZip(float* address, (Vector128<float> Value1, Vector128<float> Value2, Vector128<float> Value3) value);
++            public unsafe static void StoreVectorAndZip(float* address, (Vector128<float> Value1, Vector128<float> Value2) value);
++            public unsafe static void StoreVectorAndZip(ushort* address, (Vector128<ushort> Value1, Vector128<ushort> Value2, Vector128<ushort> Value3, Vector128<ushort> Value4) value);
++            public unsafe static void StoreVectorAndZip(ushort* address, (Vector128<ushort> Value1, Vector128<ushort> Value2, Vector128<ushort> Value3) value);
++            public unsafe static void StoreVectorAndZip(ushort* address, (Vector128<ushort> Value1, Vector128<ushort> Value2) value);
++            public unsafe static void StoreVectorAndZip(uint* address, (Vector128<uint> Value1, Vector128<uint> Value2, Vector128<uint> Value3, Vector128<uint> Value4) value);
++            public unsafe static void StoreVectorAndZip(uint* address, (Vector128<uint> Value1, Vector128<uint> Value2, Vector128<uint> Value3) value);
++            public unsafe static void StoreVectorAndZip(uint* address, (Vector128<uint> Value1, Vector128<uint> Value2) value);
++            public unsafe static void StoreVectorAndZip(ulong* address, (Vector128<ulong> Value1, Vector128<ulong> Value2, Vector128<ulong> Value3, Vector128<ulong> Value4) value);
++            public unsafe static void StoreVectorAndZip(ulong* address, (Vector128<ulong> Value1, Vector128<ulong> Value2, Vector128<ulong> Value3) value);
++            public unsafe static void StoreVectorAndZip(ulong* address, (Vector128<ulong> Value1, Vector128<ulong> Value2) value);
+         }
+     }
+     public abstract class Sve : AdvSimd {
```

Comment:
It's not new in this preview, but I want to explicitly call out that `Sve` is marked `Experimental` and will ship that way in .NET 9 (as an FYI to anyone who looks at these release notes)